client/components/organisms/SidePanel/SidePanel.tsx

```diff
@@ -65,7 +65,7 @@ const SidePanel = () => {
   return (
     <>
       <div
-        className="hidden w-16 fixed top-[60px] left-0 z-50 scroll md:flex flex-col overflow-y-auto xl:w-60 px-2 bg-white"
+        className="hidden w-16 fixed top-[70px] left-0 z-50 scroll md:flex flex-col overflow-y-auto xl:w-60 px-2 bg-white"
```

Comment:
Refactor this: don't change the top positioning as it may affect the side panel overlay